npm/webpack-dev-server/src/helpers/nextHandler.ts

```diff
@@ -131,13 +132,49 @@ function getNextJsPackages (devServerConfig: WebpackDevServerConfig) {
       middlewareMatchers?: MiddlewareMatcher[]
     }
   ]
+
+ * - v13.2.1
+  [
+    dir: string,
+    options:  {
+    buildId: string
+    config: NextConfigComplete
+    compilerType: CompilerNameValues
+    dev?: boolean
+    entrypoints: webpack.EntryObject
+    isDevFallback?: boolean
+    pagesDir?: string
+    reactProductionProfiling?: boolean
+    rewrites: CustomRoutes['rewrites']
+    runWebpackSpan: Span
+    target?: string
+    appDir?: string
+    middlewareMatchers?: MiddlewareMatcher[]
+    noMangling?: boolean
+    jsConfig: any
+    resolvedBaseUrl: string | undefined
+    supportedBrowsers: string[] | undefined
+    clientRouterFilters?: {
+        staticFilter: ReturnType<
+          import('../shared/lib/bloom-filter').BloomFilter['export']
+        >
+        dynamicFilter: ReturnType<
+          import('../shared/lib/bloom-filter').BloomFilter['export']
+        >
+      }
+    }
+  ]
  */
 async function loadWebpackConfig (devServerConfig: WebpackDevServerConfig): Promise<Configuration> {
-  const { loadConfig, getNextJsBaseWebpackConfig } = getNextJsPackages(devServerConfig)
+  const { loadConfig, getNextJsBaseWebpackConfig, nextLoadProjectInfo } = getNextJsPackages(devServerConfig)
 
   const nextConfig = await loadConfig('development', devServerConfig.cypressConfig.projectRoot)
   const runWebpackSpan = getRunWebpackSpan(devServerConfig)
   const reactVersion = getReactVersion(devServerConfig.cypressConfig.projectRoot)
+  // If nextLoadProjectInfo is not a function, then the Next.js version is < 13.2.1
+  const projectInfo = typeof nextLoadProjectInfo === 'function' ?
+    await nextLoadProjectInfo({ dir: devServerConfig.cypressConfig.projectRoot, config: nextConfig, dev: true }) :
+    undefined
```

Comment:
```suggestion
const projectInfo = await nextLoadProjectInfo?.({ dir: devServerConfig.cypressConfig.projectRoot, config: nextConfig, dev: true })
```

Could one line it with optional access